File: KantechGatewayApp/Jobs/ResigneeProcessor.vb
Namespace KantechGatewayApp.Jobs
    Public Class ResigneeProcessor
        Inherits BaseProcessor
        Protected Overrides Sub HandleRow(row As Dictionary(Of String, String))
            Infrastructure.Logger.Info($"[Resignee] card={row.GetValueOrDefault("Card ID")} emp={row.GetValueOrDefault("employee ID")} -> access removed", _logSub)
        End Sub
    End Class
End Namespace

File: KantechGatewayApp/Jobs/ResigneeProcessor.cs
using System.Collections.Generic;

namespace KantechGatewayApp.KantechGatewayApp.Jobs
{
    public class ResigneeProcessor : BaseProcessor
    {
        protected override void HandleRow(Dictionary<string, string> row)
        {
            Infrastructure.Logger.Info($"[Resignee] card={row.GetValueOrDefault("Card ID")} emp={row.GetValueOrDefault("employee ID")} -> access removed", _logSub);
        }
    }
}